libraries/wallet/wallet.cpp

```diff
@@ -2228,7 +2232,7 @@ fc::ecc::private_key wallet_api::derive_private_key(const std::string& prefix_st
             signed_transaction tx;
             chain_properties_update_operation op;
             chain_api_properties ap;
-            chain_properties_18 p;
+            chain_properties p;
```

Comment:
You break compatibility with old nodes with daemon version less HF19.
Return back the previous implementation.